src/main/java/com/denizenscript/depenizen/bukkit/clientizen/ClientizenBridge.java

```diff
@@ -0,0 +1,105 @@
+package com.denizenscript.depenizen.bukkit.clientizen;
+
+import com.denizenscript.denizen.Denizen;
+import com.denizenscript.denizen.events.bukkit.ScriptReloadEvent;
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizencore.DenizenCore;
+import com.denizenscript.denizencore.events.ScriptEvent;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptHelper;
+import com.denizenscript.denizencore.utilities.CoreConfiguration;
+import com.denizenscript.denizencore.utilities.CoreUtilities;
+import com.denizenscript.denizencore.utilities.debugging.Debug;
+import com.denizenscript.depenizen.bukkit.Depenizen;
+import com.denizenscript.depenizen.bukkit.clientizen.commands.ClientRunCommand;
+import com.denizenscript.depenizen.bukkit.clientizen.network.Channels;
+import com.denizenscript.depenizen.bukkit.clientizen.network.DataSerializer;
+import com.denizenscript.depenizen.bukkit.clientizen.network.NetworkManager;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerQuitEvent;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.util.*;
+
+public class ClientizenBridge implements Listener {
+
+    public static final Map<String, String> clientizenScripts = new HashMap<>();
+    public static DataSerializer scriptsSerializer;
+    public static final Set<UUID> clientizenPlayers = new HashSet<>();
```

Comment:
Should probably not be `final` unless there's a specific reason to be

Reply:
Removed